css/style.css

```
@@ -80,35 +110,69 @@ img {
 	background-repeat: no-repeat;
 }
 
-.container_logo h2 {
-	color: var(--basic-dark);
-	font-size: 21px;
-	line-height: 26px;
-	font-weight: bold;
+.container_img {
+	margin: 77px auto;
 }
 
-.container p {
+.reason p {
+	margin: 16px auto 51px;
 	color: var(--basic-dark);
 	font-size: 14px;
 	line-height: 26px;
 	font-weight: normal;
 }
 
-/*---------------------------------advantages---------------------------------*/
+.reason h2 {
+	width: 470px;
+	min-height: 80px;
+	color: var(--basic-dark);
+	font-size: 21px;
+	line-height: 26px;
+	font-weight: bold;
+	margin: 42px auto 0;
+}
 
+/*---------------------------------advantages---------------------------------*/
 
 .advantages_list, .footer_lists {
 	list-style: none;
 }
 
+.advantages_list {
+	display: grid;
+	grid-template-rows: 1fr 1fr 1fr;	/*Упростить*/
+}
+
+.advantages_items_first {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	grid-template-rows: 1fr 1fr;
+}
+
+.advantages_items_mountains {
+	grid-column: 2/4;
+	grid-row: 1/2;
+}
+
+.advantages_items_second {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
```

```suggestion
	grid-template-columns: 1fr 1fr;
```
А также блоку нужно добавить свойство `rid-column: 1 / -1`.